pretix_passbook/passbook.py

```diff
@@ -132,6 +133,42 @@ def settings_form_fields(self) -> dict:
                      ),
                      required=False,
                  )),
+                (
+                    'bg_color',
+                    forms.CharField(
+                        label=_('Background color'),
```

Comment:
Should we add a help text along the lines of "The background color will be ignored if a background image is uploaded"?

Reply:
Good idea! I added a help text.